applications/plugins/SofaTest/AffinePatch_test.cpp

```diff
@@ -49,7 +49,7 @@ using namespace modeling;
 An affine movement (rotation and translation) is applied to the borders of a mesh. Test if the points inside have the same affine movement.*/
 
 template <typename _DataTypes>
-struct AffinePatch_test : public Elasticity_test<_DataTypes>
+struct AffinePatch_sofa_test : public Elasticity_test<_DataTypes>
```

Comment:
Why this?

Reply:
because there is a test in Flexible  with exactly the same name, hence messing up

Reply:
too many tests in SOFA!!